pquery-run.sh

```diff
@@ -1388,24 +1399,36 @@ pquery_test(){
           else
             echoit "Loading metadata from ${WORKDIR}/step_$((${TRIAL}-1)).dll ..."
           fi
+          if [[ ${PXC} -eq 0 && ${GRP_RPL} -eq 0 ]]; then
+            CMD="${PQUERY_BIN} ${SQL_FILE} --database=test --threads=${THREADS} --queries-per-thread=${QUERIES_PER_THREAD} --logdir=${RUNDIR}/${TRIAL} --log-all-queries --log-failed-queries --user=root --socket=${SOCKET} --seed ${SEED} --step ${TRIAL} --metadata-path ${WORKDIR}/ --seconds ${PQUERY_RUN_TIMEOUT} ${DYNAMIC_QUERY_PARAMETER}"
```

Comment:
can you remove SQL_FILE if not consumed by pquery

Reply:
--log-all-queries, --log-failed-queried make it configurable.

Reply:
SQL_FILE option is for pquery2, no use for pquery3. Hence removed.
--log-all-queries, --log-failed-queries have been moved to the configuration file and removed from pquery-run.sh